### ISSUE TITLE
Selecting/Copying/Pasting Regex occurrences

### ISSUE THREAD

**Issue by FilipeD** [microsoft/vscode#107018]
Issue Type: <b>Bug</b>

1)find whatever you want with a regex
2)select all occurrences
3)edit->copy selection
4)change to another tab
5)paste previously copied selection

Expected behaviour (for sake of brevity), here's a few dates:
em 27-IX-1853
em 27-IX-1932
em 17-VII-1913
em 30-VI-1932

If I try to find \d{4} and select all occurrences via menu or shortcut ctrl-shift-L, after copying either via edit-copy or ctrl-c
and moving to another tab, I expect to get: 
1853
1932
1913
1932

Actual outcome when pasting:
whatever-was-on-clipboard-before-last-attempt-to-copy

The point is: one has to perform the "select all occurences + copy + paste"
sequence TWICE(!!!) in order to actually have the selected occurrences copied and pasted.

VS Code version: Code 1.49.0 (e790b931385d72cf5669fcefc51cdf65990efa5d, 2020-09-10T13:22:08.892Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz (8 x 3408)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|30.91GB (11.30GB free)|
|Process Argv|--crash-reporter-id 2aaa405c-144e-4c8b-9b98-ca5048684c17|
|Screen Reader|no|
|VM|36%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|2.1.8
prettier-vscode|esb|5.6.0
vscode-JS-CSS-HTML-formatter|lon|0.2.3
Kotlin|mat|1.7.1
rainbow-csv|mec|1.7.1
python|ms-|2020.8.109390
cpptools|ms-|1.0.0
JavaScriptSnippets|xab|1.8.0
vscode-proto3|zxh|0.4.2

</details>

**Comment by alexdima** [comment]
@FilipeD I cannot reproduce:

![Kapture 2020-09-18 at 21 07 31](https://user-images.githubusercontent.com/5047891/93635688-02595700-f9f3-11ea-9e1e-66b8d988fba7.gif)

Could you please try to disable your extensions? Press `F1` and then choose `Developer: Reload Window with Extensions Disabled`

![Kapture 2020-07-20 at 15 28 23](https://user-images.githubusercontent.com/5047891/87943040-c3b02a80-ca9d-11ea-847d-5f850fae8f05.gif)

**Comment by FilipeD** [comment]
Hi, sorry for the delay...
Back to work, tried reloading as you asked, no effect.
You're on a MAC :)
The clipboard and shortcuts are different so... Must be a Windows-only implementation issue?
Please have a look at the attached mp4 (and ignore the last few seconds after typing "only works on 2nd attempt").
[VSCode copy Issues.zip](https://github.com/microsoft/vscode/files/5276568/VSCode.copy.Issues.zip)

**Comment by alexdima** [comment]
Ok. Here is what is going on. You click on a file in the explorer. The focus is on a file called `brutoUTF16LE.txt`. You then go to the menu and run `Selection > Select All Occurrences`. This selects all occurrences, as advertised, but it **does not move the focus to the editor**. The focus goes back to the explorer, to the file called `brutoUTF16LE.txt`. You execute `Edit > Copy`. This does not overwrite the textual clipboard because the file is remembered in an alternative clipboard. You then paste, so you paste what you copied previously.

So long story short, `Edit > Copy` will execute the Copy action on whatever happens to be focused. Focus was not in the editor.
